src/azure-cli/azure/cli/command_modules/sql/tests/latest/test_sql_commands.py

```diff
@@ -2987,6 +2990,39 @@ def test_sql_managed_instance_mgmt(self):
                      JMESPathCheck('resourceGroup', resource_group_1),
                      JMESPathCheck('minimalTlsVersion', tls1_1)])
 
+        # test update managed instance tags
+        tag3 = "tagName3=tagValue3"
+        self.cmd('sql mi update -g {} -n {} --set tags.{}'
+                 .format(resource_group_1, managed_instance_name_1, tag3),
```

Comment:
Please recording the test for your test cases. I haven't seen recording yaml file.

Reply:
remove test_sql_managed_instance_mgmt.yaml in recording and run `azdev test test_sql_managed_instance_mgmt`

Reply:
My IP was blacklisted on azure because thousands of concurrent calls to GET  management.azure.com/subscriptions API coming from my IP address. Because of that, when I try running tests in record mode they fail. We tried to repro the issue on my machine, buth without success. Do you know whether setting environment for CLI could have caused this burst of API calls because that was the only thing I did that day that I don't usually do?

Reply:
I've just pushed a new commit with recording.